src/client/src/client_main.cpp

```diff
@@ -0,0 +1,59 @@
+//
+// Created by kevin on 6/25/22.
+//
+#include <ctime>
+#include <cstring>
+#include <ros/ros.h>
+#include <client/current_time.h>
+#include <client/login.h>
+#include <iostream>
+
+using namespace std;
+
+ros::ServiceClient service_client;
+
+int main(int argc, char **argv) {
+    string main_name;
+    bool logged = 0;
+    cin>>main_name;
+    ros::init(argc, argv, main_name);
+    ros::NodeHandle client_node_handle;
+
+    client::current_time publishing_msg;
+    ros::Publisher client_publisher = client_node_handle.advertise<client::current_time>(main_name, 1);
+
+    service_client = client_node_handle.serviceClient<client::login>("ros_learn_login_service");
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]